github/ci/services/prometheus-stack/patches/stack/production-e2e-workloads/prometheus.yaml

```diff
@@ -26,6 +26,15 @@ spec:
           matchLabels:
             app: prometheus
             type: local
+  additionalScrapeConfigs:
```

Comment:
Same as above, where does this `additionalScrapeConfigs` come from?

Reply:
As I commented before, it was in the original kube-prometheus-stack.yaml files